.copr/make-srpm.sh

```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
-set -ex
+set -eux
 
 outdir="$1"; shift
 
-dirname="$(dirname "$0")"
+rootdir="$(realpath -m "$0"/../..)"
 
 DISTGIT_URL=https://src.fedoraproject.org/rpms/selinux-policy
 DISTGIT_REF=rawhide
 
 CONTAINER_URL=https://github.com/containers/container-selinux
 EXPANDER_URL=https://github.com/fedora-selinux/macro-expander
 
-rpm -q rpm-build git-core || dnf install -y rpm-build git-core
+rpm -q rpm-build git-core
```

Comment:
Ah, ok -- I was going to throw it out as the next step anyway, when moving the whole build-rpms workflow and copr handling to packit (I talked to @zpytela about it yesterday). See https://github.com/martinpitt/selinux-policy/pull/1 . I'm happy to revert this, it's not important at all.

Reply:
I'd prefer to keep `.copr/` in the repo. I use it when I have a branch with some testing changes in a branch on my fork - having the `.copr` directory there allows me to set up automatic COPR builds without needing to add any special changes. I would like this to be preserved along with the Packit support. I guess the `make-srpm.sh` script will need to be massaged a bit to be usable for both, but hopefully it won't be too complicated.

There is also https://copr.fedorainfracloud.org/coprs/omos/selinux-policy-latest that is set to build the latest rawhide branch content after every push, but I think no one actually uses it (and I presume it would be made obsolete by the Packit support).

Reply:
Packit COPRs also "just work" -- I forked the project and sent https://github.com/martinpitt/selinux-policy/pull/1 (to my own fork, as it's building on top of this), and if you follow the "rpm-build*" test contexts, you get to https://dashboard.packit.dev/results/copr-builds/912150 -- i.e. that just effortlessly works on forks as well, which is great for experiments and collaboration. That said, I can rework this to work in both cases, maybe with splitting up the script.

Your omos/selinux-policy-latest copr, you said "is set to" -- does copr itself provide automatic builds? If so, that will continue to work of course (I wasn't aware that COPR can do that). Otherwise we can add a [copr on-commit job](https://github.com/cockpit-project/cockpit-podman/blob/82329ce4e71b644815d6df61fa0ed571fddc9526/packit.yaml#L54) to do that, provided that you allow the "packit" user as a builder in your copr.

Reply:
> Not a copr, but scratchbuilds are available at Checks -> Artifacts -> rpms.zip - is that what you meant?

@zpytela : No, that would be builds from the proposed branches, not after the merge to the main branch ("rawhide" or whichever the target is). I meant the kind of COPR that @WOnder93 mentioned [above](https://github.com/fedora-selinux/selinux-policy/pull/1812#discussion_r1282000777).

Reply:
Note that this discussion isn't really relevant for *this* PR, but very much so for the next one that I'm going to send after this lands :grin: so thanks!

Reply:
FTR: I understand that you are cautious here. I'll rework the "packit copr" PR to keep `.copr/` functional as-is, and instead provide an option for make-srpm.sh to skip the rpm-build step.